### PR TITLE
Fix Unit Tests

### PR DESCRIPTION
**Pull request by shulkaolka**

### CODE REVIEW

File: spec/common.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ process.env.ELASTICIO_API_USERNAME = 'user';
 process.env.ELASTICIO_API_KEY = 'apiKey';
 process.env.ELASTICIO_WORKSPACE_ID = 'workspaceId';
 
-const EXT_FILE_STORAGE = 'http://file.storage.server/file';
+const EXT_FILE_STORAGE = 'http://file.storage.server';
```
Comment (denyshld) [inline spec/common.js:10]: Looks like logic change

File: spec/testData/bulk_q.json [hunk 2]
```diff
@@ -40,9 +40,9 @@
         }
       },
       "http://file.storage.server": {
-        "/file": {
+        "/": {
           "method": "PUT",
-          "response": "OKs"
```
Comment (denyshld) [inline spec/testData/bulk_q.json:45]: Looks like processing logic change

File: spec/common.js (hunk 1 above)
Comment (shulkaolka) [reply spec/common.js:10]: Was changed for proper work `nock` library (the library was updated)

File: spec/testData/bulk_q.json (hunk 2 above)
Comment (shulkaolka) [reply spec/testData/bulk_q.json:45]: accidentally was changed in the previous branch. Now reverted to the proper value - see master branch https://github.com/elasticio/salesforce-component/blob/master/spec/actions/bulk_q.json#L45

**Review by denyshld — APPROVED** [review]

**Pull request merged**